miniwin/src/d3drm/backends/opengles3/renderer.cpp

```diff
@@ -626,67 +622,36 @@ void OpenGLES3Renderer::Resize(int width, int height, const ViewportTransform& v
 	m_renderedImage = SDL_CreateSurface(m_width, m_height, SDL_PIXELFORMAT_RGBA32);
 
 	if (m_colorTarget) {
-		glDeleteTextures(1, &m_colorTarget);
+		glDeleteRenderbuffers(1, &m_colorTarget);
+		m_depthTarget = 0;
 	}
 	if (m_depthTarget) {
 		glDeleteRenderbuffers(1, &m_depthTarget);
+		m_colorTarget = 0;
 	}
-	if (m_msaaColorRbo) {
-		glDeleteRenderbuffers(1, &m_msaaColorRbo);
-	}
-	if (m_msaaDepthRbo) {
-		glDeleteRenderbuffers(1, &m_msaaDepthRbo);
-	}
-	m_colorTarget = m_depthTarget = m_msaaColorRbo = m_msaaDepthRbo = 0;
-
-	GLint samples, maxSamples;
-	glGetIntegerv(GL_SAMPLES, &samples);
-	glGetIntegerv(GL_MAX_SAMPLES, &maxSamples);
-	m_useMsaa = samples > 1;
-
-	if (m_useMsaa) {
-		glBindFramebuffer(GL_FRAMEBUFFER, m_msaaFbo);
-
-		glGenRenderbuffers(1, &m_msaaColorRbo);
-		glBindRenderbuffer(GL_RENDERBUFFER, m_msaaColorRbo);
-		glRenderbufferStorageMultisample(GL_RENDERBUFFER, samples, GL_RGBA8, width, height);
-		glFramebufferRenderbuffer(GL_FRAMEBUFFER, GL_COLOR_ATTACHMENT0, GL_RENDERBUFFER, m_msaaColorRbo);
-
-		glGenRenderbuffers(1, &m_msaaDepthRbo);
-		glBindRenderbuffer(GL_RENDERBUFFER, m_msaaDepthRbo);
-		glRenderbufferStorageMultisample(GL_RENDERBUFFER, samples, GL_DEPTH24_STENCIL8, width, height);
-		glFramebufferRenderbuffer(GL_FRAMEBUFFER, GL_DEPTH_ATTACHMENT, GL_RENDERBUFFER, m_msaaDepthRbo);
-
-		if (glCheckFramebufferStatus(GL_FRAMEBUFFER) != GL_FRAMEBUFFER_COMPLETE) {
-			SDL_Log("MSAA Framebuffer is not complete! Disabling MSAA.");
-			m_useMsaa = false;
-		}
-	}
-
-	SDL_Log(
```

Comment:
Can we keep some kind of logging? Otherwise it's hard to tell what is active and what isn't